### PR TITLE
Tarea 3

### PR DESCRIPTION
**Pull request by OlmeiroOrozco**

### CODE REVIEW

File: olmeiro-orozco/tarea-3/OddOrEven.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// Input: [0, 1, 4]
+// Output: "odd"
+
+// Input: [0, -1, -5]
+// Output: "even"
+
+function oddOrEven(array) {
+  let suma = 0;
+  // console.log(array);
+  if (array.length === 1 && array[0] === 0) {
```
Comment (emolinaglobant) [inline olmeiro-orozco/tarea-3/OddOrEven.js]: @OlmeiroOrozco , you may have the third condition in this one

```suggestion
  if (array.length === 0 || (array.length === 1 && array[0] === 0)) {
```

File: olmeiro-orozco/tarea-3/OddOrEven.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    return "even";
+  } else if (array.length === 0 && array[0] === 1) {
+    return "odd";
+  } else if (array.length === 0) {
+    return "even";
+  } else {
+    suma = array.reduce((acc, n) => acc + n);
+    // console.log("suma: ", suma);
+    if (suma % 2 === 0) {
+      return "even";
```
Comment (emolinaglobant) [inline olmeiro-orozco/tarea-3/OddOrEven.js]: @OlmeiroOrozco , is not a good practice have so many return sentences , you should use a result variable and only one return at the end

File: olmeiro-orozco/tarea-3/WhiteOrBlack.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  let oddFiles = ["a", "c", "e", "g"];
+  let oddRanks = [1, 3, 5, 7];
+  let evenFiles = ["b", "d", "f", "h"];
+  let evenRanks = [2, 4, 6, 8];
+
+  if (oddFiles.includes(file)) {
+    //esimpar
+    if (oddRanks.includes(rank)) {
+      //es impar
+      return "black";
```
Comment (emolinaglobant) [inline olmeiro-orozco/tarea-3/WhiteOrBlack.js]: @OlmeiroOrozco , is not a good practice have so many return sentences , you should use a result variable and only one return at the end

**Review by emolinaglobant — CHANGES_REQUESTED** [review]

File: olmeiro-orozco/tarea-3/OddOrEven.js (hunk 1 above)
Comment (OlmeiroOrozco) [reply olmeiro-orozco/tarea-3/OddOrEven.js]: I did the change.

File: olmeiro-orozco/tarea-3/OddOrEven.js (hunk 2 above)
Comment (OlmeiroOrozco) [reply olmeiro-orozco/tarea-3/OddOrEven.js]: I did the change.

File: olmeiro-orozco/tarea-3/WhiteOrBlack.js (hunk 3 above)
Comment (OlmeiroOrozco) [reply olmeiro-orozco/tarea-3/WhiteOrBlack.js]: I did the change.

**Review by emolinaglobant — APPROVED** [review]

**Pull request merged**